lntypes/txsize_test.go

```diff
@@ -0,0 +1,28 @@
+package lntypes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestTxSizeUnit tests the conversion of tx size to different units.
+func TestTxSizeUnit(t *testing.T) {
+	t.Parallel()
+
+	// Test normal conversion from 100wu to 25vb.
+	wu := WeightUnit(100)
+	vb := VByte(25)
+	require.Equal(t, vb, wu.ToVB(), "wu -> vb conversion "+
```

Comment:
nit: need to use `Equalf()` if there are format placeholders. Though I don't think we need the "want / got" part, as the `require` library already provides that. So just `"wu -> vb conversion"` as the context argument IMO is enough. Also allows this to be a one-liner.

Reply:
yeah I think `require` has been updated and now `Equal` and `Equalf` are the same now...didn't realize this until you mentioned it here. In addition I'm not good at reading hex decimals, so kinda like the "want/got", which looks like,
```
--- FAIL: TestTxSizeUnit (0.00s)
    txsize_test.go:24: 
        	Error Trace:	.../lnd/lntypes/txsize_test.go:24
        	Error:      	Not equal: 
        	            	expected: 0x1a
        	            	actual  : 0x19
        	Test:       	TestTxSizeUnit
        	Messages:   	wu -> vb conversion failed: want 26 vb, got 25 vb
```

Reply:
Ah, okay, good point with the hex values, didn't think about that.